Dockerfile

```diff
@@ -200,8 +200,8 @@ RUN apk add \
     /venv/bin/python3 -m pip install --no-cache-dir gunicorn==20.1.0 && \
     find / -type d -name '__pycache__' -exec rm -rf {} + && \
     useradd -r -u 5050 -g root -s /sbin/nologin pgadmin && \
-    mkdir -p /var/lib/pgadmin && \
-    chown pgadmin:root /var/lib/pgadmin && \
+    mkdir -p /run/pgadmin /var/lib/pgadmin && \
```

Comment:
Do you think this will work? It will create /run/pgadmin directory only, not the /var/lib/pgadmin.

Reply:
The creation and rights of the `/var/lib/pgadmin` folder is in the original Dockerfile, my modifications are only adding `/run/pgadmin` folder and needed rights for that commands.

I'm not running `pgadmin4` Dockerfile in my project directly, I'm importing it as a base image, I cannot test it right now. I suggest leaving it there for later testing, than it can be removed or not.

Reply:
I tested the image now, both folder created without problem. `mkdir` can take multiple arguments for directory(ies), In my images I use it this way in `debian` and `alpine` as well:

* [MAN page](https://linux.die.net/man/1/mkdir)
* [BusyBox coreutils `mkdir` source](https://github.com/mirror/busybox/blob/2d4a3d9e6c1493a9520b907e07a41aca90cdfd94/coreutils/mkdir.c#L100)

I run Docker in Windows+WSL2 Ubuntu, if it's not creating both folders for everyone, perhabs good idea to separate the commands? Have you tested it on your side?